app/scripts/components/common/mapbox/map.tsx

```diff
@@ -38,6 +38,7 @@ interface SimpleMapProps {
   onAoiChange?: AoiChangeListenerOverload;
   projection?: ProjectionOptions;
   onProjectionChange?: (projection: ProjectionOptions) => void;
+  attributionPosition?: 'top-right' | 'top-left' | 'bottom-right' | 'bottom-left' | false
```

Comment:
It's a shame this is not defined as a reusable independent `AttributionPosition` type in the Mapbox GL typings. Worth opening a PR [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/mapbox-gl/index.d.ts#L207)?